docs/conf.py

```diff
@@ -19,6 +19,7 @@
 # absolute, like shown here.
 sys.path.append(path.join(path.dirname(__file__), "_ext"))
 sys.path.append(path.join(path.dirname(path.dirname(__file__)), "scrapy"))
```

Comment:
Also, I'm confused what this line is for? Are any Scrapy submodules ever referenced without `scrapy.` prefixed? Removing it creates no errors on my system.